src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -1493,12 +1493,16 @@ void Instruction::execute() {
         return executionNYI();
         break;
       }
-      case Opcode::AArch64_CMEQv8i8: {
-        return executionNYI();
+      case Opcode::AArch64_CMEQv8i8: {  // cmeq vd.8b, vn.8b, vm.8b
+        results[0] = neonHelp::vecCompare<uint16_t, 8>(
+            operands, false,
+            [](uint32_t x, uint32_t y) -> bool { return (x == y); });
```

Comment:
The lambda should take in type `uint8_t` rather than `uint32_t` to match the operand type used by the helper funtion.